charts/csi-unity/values.yaml

```diff
@@ -242,6 +242,20 @@ maxUnityVolumesPerNode: 0
 # Examples : "tenant2" , "tenant3"
 tenantName: ""
 
+# Storage Capacity Tracking
+# Note: Capacity tracking is supported in kubernetes v1.24 and above, this feature will be automatically disabled in older versions.
+storageCapacity:
```

Comment:
Add this parameter in container-storage-modules/values,yaml as well and make the necessary changes in the Installation Wizard

Reply:
Taken care